scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -115,8 +115,9 @@ public void build() {
         this.setSelectionType(SelectionStyle.SINGLE);
         this.setSelectionProperty("isSelected");
         this.sort(TasksCentricColumnsFactory.JOB_ID_ATTR.getName(), SortDirection.DESCENDING);
-        this.setShowRecordComponents(true);
-        this.setShowRecordComponentsByCell(true);
+        //        These lines break task centric scrolling
+        //        this.setShowRecordComponents(true);
```

Comment:
Why keeping those lines as comment ?

Reply:
Because code can be in comments. It shows previous version which we might still need in the future.